content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -1,79 +1,27 @@
 ---
 reviewers:
-- mikedanese
-- luxas
-- errordeveloper
-- jbeda
-title: Using kubeadm to Create a Cluster
+- sig-cluster-lifecycle
+title: Creating a single master cluster with kubeadm
 content_template: templates/task
 ---
 
 {{% capture overview %}}
 
-<img src="https://raw.githubusercontent.com/cncf/artwork/master/kubernetes/certified-kubernetes/versionless/color/certified-kubernetes-color.png" align="right" width="150px">**kubeadm** is a toolkit that helps you bootstrap a best-practice Kubernetes
-cluster in an easy, reasonably secure and extensible way. It also supports
-managing [Bootstrap Tokens](/docs/admin/bootstrap-tokens/) for you and upgrading/downgrading clusters.
+<img src="https://raw.githubusercontent.com/cncf/artwork/master/kubernetes/certified-kubernetes/versionless/color/certified-kubernetes-color.png" align="right" width="150px">**kubeadm** helps you bootstrap a minimum viable Kubernetes cluster that conforms to best practices.  With kubeadm, your cluster should pass [Kubernetes Conformance tests.](https://kubernetes.io/blog/2017/10/software-conformance-certification) Kubeadm also supports other cluster 
+lifecycle functions, such as upgrades, downgrade, and managing [Bootstrap Tokens.](/docs/admin/bootstrap-tokens/) 
 
-kubeadm aims to set up a minimum viable cluster that pass the
-[Kubernetes Conformance tests](https://kubernetes.io/blog/2017/10/software-conformance-certification), but installing other addons than
-really necessary for a functional cluster is out of scope.
-
-It by design does not install a networking solution for you, which means you
-have to install a third-party CNI-compliant networking solution yourself
-using `kubectl apply`.
-
-kubeadm expects the user to bring a machine to execute on, the type doesn't
-matter, can be a Linux laptop, virtual machine, physical/cloud server or
-Raspberry Pi. This makes kubeadm well suited to integrate with provisioning
-systems of different kinds (e.g. Terraform, Ansible, etc.).
+Because you can install kubeadm on any type of machine (e.g. laptop, server, 
+Raspberry Pi, etc.), it's well suited to integration with provisioning systems 
```

Comment:
s/to integration/for integration/

Might also want to add other CM tools such as Puppet and Chef here as well. It might also be worth mentioning that it could be scripted to run from cloud-init as well.

Reply:
FYI #8910.

Currently we have some limitations.